VespaGAS/Managers/State/OilMixState.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
```

Comment:
The application state is defined in a single data structure. The idea is that this struct should store the ENTIRE state of your app 😓